### PR TITLE
Issue #534: upgrade Ubuntu to 20.04 for CI

### PR DESCRIPTION
**Pull request by pbludov**
Issue #534

I'm already using this distro on my PC, so no issues are expected.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,43 +8,43 @@ install:
 matrix:
   fast_finish: true
   include:
-    - jdk: openjdk8
+    - jdk: openjdk11
```
Comment (romani) [inline .travis.yml:11]: I am not sure 😬 it good to drop jdk8 build validation.
Issue is about to please me linter.

**Review by romani — CHANGES_REQUESTED** [review]
Item:

File: .travis.yml (hunk 1 above)
Comment (pbludov) [reply .travis.yml:11]: Ubuntu 20.04 in Travis CI has no support for openjdk8. Only openjdk 9+

Comment (romani) [reply .travis.yml:11]: What if we activate azure ?
To make it like https://github.com/checkstyle/checkstyle/blob/74eb9219b6892f288cd293162666643b811ff5ff/azure-pipelines.yml#L117

Comment (pbludov) [reply .travis.yml:11]: Yes, this should be azure pipiline. It is better to have common CI for all projects.

**Comment by rnveach** [comment]
Until this is fixed, CI is of no use in this repo.

File: .travis.yml (hunk 1 above)
Comment (romani) [reply .travis.yml]: we have problems to share permission to azure project/organization ....
we can not come to agreement to [migrate to jdk11](https://github.com/checkstyle/checkstyle/issues/9146) in main repo ....

as this repo is not a library! jdk consideration are valid only for libraries. It is unlikely that we will use some jdk syntax feature so quickly.
I am completely ok to upgrade Travis to new OS version and use jdk11 for all builds in contribution repo.

Lets keep Azure as backup (second execution of the same), we need to comeback to this, but at least we will have config already.

**Comment by pbludov** [comment]
closing for now, we can return to this issue after migration to java11 as the minimal version.

**Pull request closed without merging**